app/helpers/constants.rb

```diff
@@ -0,0 +1,10 @@
+module Constants
+  CATALOG_BOOK_COUNT = 3
+  BEST_SELLER_COUNT = 4
+
+  GREATER_THEN_ORT_EQUAL_TO = 0.0
+  LESS_THEN_OR_EQUAL_TO = 10_000
```

Comment:
Wrong naming. What is the purpose of these numbers? If it price, then rename `MIN_PRICE/MAX_PRICE`